robosuite/devices/device.py

```diff
@@ -163,10 +163,11 @@ def input2action(self, mirror_actions=False) -> Optional[Dict]:
         )
         ac_dict[f"{active_arm}_abs"] = arm_action["abs"]
         ac_dict[f"{active_arm}_delta"] = arm_action["delta"]
-        ac_dict[f"{active_arm}_gripper"] = np.array([grasp] * gripper_dof)
+        # ac_dict[f"{active_arm}_gripper"] = np.array([grasp] * gripper_dof)
+        ac_dict[f"{active_arm}_gripper"] = [1, 0.45 * grasp, grasp, grasp, grasp, grasp]
```

Comment:
what does this part change?

> 0.45 * grasp

this seems specific to the Inspire hand. We should make it a case by case basis
Also not sure if it works for non-humanoids

Reply:
Agreed. This would impact other robots. Moved the changes into class `InspireLeftHand` and `InspireRightHand`